messages.json

```diff
@@ -0,0 +1,16 @@
+{
+  "prompts": [
+    {
+      "type": "question",
+      "message": "Hello world",
+      "channelID": 1234,
+      "intervalSeconds": 10
```

Comment:
Nice! I like the idea of having the channel id as a field.

Could we make this snake_case to be consistent with Python naming though?
- `channelId` -> `channel_id`
- `intervalSeconds` -> `interval_seconds`

Same for the prompt below.